client/src/actions/mealsActions.js

```diff
@@ -9,6 +9,7 @@ export const FETCH_MEALDETAIL = 'FETCH_MEALDETAIL';
 export const FETCH_MEALS_BY_CHEF = 'FETCH_MEALS_BY_CHEF';
 export const GET_NEAR_BY_MEALS = 'GET_NEAR_BY_MEALS';
 export const FETCH_TODAYS_MEALS = 'FETCH_TODAYS_MEALS';
+export const FETCH_UPCOMING_MEALS = 'FETCH_UPCOMING_MEALS';
```

Comment:
I would delete fetch today and tomorrow's meals logic (reducers and actions creators) bc we are not going to use them anymore